tests/test_edgeql_data_migration.py

```diff
@@ -7930,6 +7979,36 @@ async def test_edgeql_migration_prompt_id_01(self):
             },
         })
 
+    async def test_edgeql_migration_user_input_01(self):
+        await self.migrate('''
+            type Bar { property foo -> str };
+        ''')
+
+        await self.start_migration('''
+            type Bar { property foo -> int64 };
+        ''')
+
+        await self.assert_describe_migration({
+            'proposed': {
+                'statements': [{
+                    'text': '''
+                        ALTER TYPE test::Bar {
+                            ALTER PROPERTY foo {
+                                SET TYPE std::int64 USING (\\(cast_expr));
```

Comment:
I'd use an `r''' ... '''` string to have single slashes here. Would make it easier to read.